src/components/searchSection/searchSection.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import {
+  MDBCol,
+  MDBRow,
+  MDBContainer,
+  MDBInput,
+  MDBIcon,
+  MDBBtn,
+  MDBFormInline,
+} from 'mdbreact';
+import './myStyle.scss';
+import CatergrayList from './catergrayList';
+
+function SearchSection() {
+  let catograysBook = [
+    { name: 'Popular' },
+    { name: 'Biographies &Memoirs' },
+    { name: 'Literature & Fiction' },
+    { name: 'Business&Investing' },
+    { name: 'History' },
+  ];
+  let AuthorNames = [
+    { name: 'zeena kareem' },
+    { name: 'zainab azzam' },
+    { name: 'sara ahmed' },
+    { name: 'dunia tarq' },
+  ];
+  let [inputValue, setInputValue] = useState('');
+  //we use number 2 as default value to make the BOOKS choosen in first time when we open the website
+  let [activeItem, setActiveItem] = useState(2);
+  let [data, setData] = useState(catograysBook);
+
+  const handelColor = (index) => {
+    if (activeItem === index) {
+      return 'textPink';
+    }
+  };
```

Comment:
same as what I said in another comment

Reply:
also there is two methods of the same thing try to use only one of them in both places